eidpauth/src/test/java/uk/ac/ucl/eidp/auth/jwt/JwtConfigNgTest.java

```diff
@@ -1,30 +1,54 @@
 package uk.ac.ucl.eidp.auth.jwt;
 
-import static org.testng.Assert.assertNull;
+import static org.testng.Assert.assertEquals;
 
 import org.testng.annotations.AfterClass;
 import org.testng.annotations.AfterMethod;
 import org.testng.annotations.BeforeClass;
 import org.testng.annotations.BeforeMethod;
 import org.testng.annotations.Test;
 
+import java.io.File;
+import java.io.FileOutputStream;
 import java.security.Key;
+import java.security.KeyStore;
+import java.util.Base64;
+import javax.crypto.KeyGenerator;
+
 
 /**
  * Unit test for JwtConfig.
  * @author David Guzman
  */
 public class JwtConfigNgTest {
   
+  private final static KeystoreProperties keystoreProperties = new KeystoreProperties();
```

Comment:
\[Checkstyle\] WARNING: 'static' modifier out of order with the JLS suggestions\.